test/visualise.js

```diff
@@ -0,0 +1,21 @@
+'use strict';
+
+import {renderToString} from "react-dom/server";
+
+module.exports = (component, path) =>
+		afterAll(() => {
+			if (process.env.VISUALISE) {
+				const html = `<html>
+			<head>
+					<title>Component Preview</title>
+					<meta name="viewport" content="width=device-width, initial-scale=1.0" />
+			</head>
+			<body style="margin:0;">
+					<div id="app">
+						 ${renderToString(component)}
+					</div>
+			</body>
+			</html>`;
```

Comment:
Would be nice to fix the indentation here.

Reply:
Oh yeh, it doesn't look like that on mine... stupid intelliJ mucking around with whitespace settings again. Fixed